apps/prairielearn/src/pages/instructorAssessmentManualGrading/assessment/assessment.ts

```diff
@@ -0,0 +1,29 @@
+import * as express from 'express';
+import asyncHandler = require('express-async-handler');
+import * as sqldb from '@prairielearn/postgres';
+import * as error from '@prairielearn/error';
+import { ManualGradingAssessment, ManualGradingQuestionSchema } from './assessment.html';
+
+const router = express.Router();
+const sql = sqldb.loadSqlEquiv(__filename);
+
+router.get(
+  '/',
+  asyncHandler(async (req, res, next) => {
+    if (!res.locals.authz_data.has_course_instance_permission_view) {
+      return next(error.make(403, 'Access denied (must be a student data viewer)'));
```

Comment:
While it does the same thing, conventionally we `throw` an error when we're inside an async handler.

```suggestion
      throw error.make(403, 'Access denied (must be a student data viewer)');
```

Reply:
Done in aae3abf1c.